.eslintrc.js

```diff
@@ -499,6 +499,7 @@ module.exports = {
 					'__experimentalItemGroup',
 					'__experimentalNavigationBackButton',
 					'__experimentalNavigatorBackButton',
+					'__experimentalNavigatorToParentButton',
```

Comment:
Since this is an experimental API, we need to disable the linter warning in the `.eslintrc.js` file so that we don’t see the `wpcalypso/no-unsafe-wp-apis` warning.